fews_3di/simulation.py

```diff
@@ -24,8 +24,9 @@
 from fews_3di import utils
 from pathlib import Path
 from threedigrid.admin.gridresultadmin import GridH5ResultAdmin
-from typing import List
 from typing import Tuple
+from typing import List
```

Comment:
Dit is niet op alfabetische volgorde => zoals in de README staat "make beautiful" draaien. Dat haalt denk ik ook wat test errors eruit.